conan/tools/scm/git.py

```diff
@@ -216,3 +216,12 @@ def included_files(self):
         files = self.run("ls-files --full-name --others --cached --exclude-standard")
         files = files.splitlines()
         return files
+
+    def to_conandata(self):
```

Comment:
Not too sold on the names of these 2, but also not a blocker either I guess :)

Reply:
I agree! :) Suggestions?